server/models/Notification.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  photo: {
+    type: String,
+    default: "",
+  },
+  type: {
+    type: String,
+    enum: ["booking", "payment", "system"],
+    lowercase: true,
+    required: true,
+  },
+  link: {
+    type: String,
+    lowercase: true,
+    required: true,
+  },
+  status: {
```

Comment:
Renaming this field to `read` and making it a boolean type seems more appropriate here